release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1014,6 +1014,24 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* Before this update, invalid subscription labels were created when a resource name exceeded 63 characters. Truncating labels that exceed the 63 character limit resolves the issue, and the subscription resource no longer rejects the Kubernetes API. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2016425[*BZ#2016425*])
+
+* Before this update, catalog source pods for the Marketplace Operator prevented nodes from draining. As a result, the Cluster Autoscaler could not scale down effectively. With this update, adding the `cluster-autoscaler.kubernetes.io/safe-to-evict` annotation to the catalog source pods fixes the issue, and the Cluster Autoscaler can scale down effectively.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2053343[*BZ#2053343*])
+
+* Before this update, the `collect-profiles` job could take a long time to complete in certain circumstances, such as when a pod could not be scheduled. As a result, if enough jobs were scheduled but unable to run, the number of scheduled jobs exceeded pod quota limits. With this update, only one `collect-profiles` pod exists at a time, and the `collect-profiles` job does not exceed pod quota limits. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055861[*BZ#2055861*])
+
+* Before this update, the package server was not aware of pod topology when defining its leader election duration, renewal deadline, and retry periods. As a result, the package server strained topologies with limited resources, such as single node environments. This update introduces a `leaderElection` package that sets reasonable lease duration, renewal deadlines, and retry periods. This fix reduces strain on clusters with limited resources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048563[*BZ#2048563*])
+
+* Previously, there was a bad catalog source in the openshift-marketplace. Because of this, all subscriptions were blocked. With this update, if there is a bad catalog source in the openshift-marketplace, users can subscribe to an operator from a quality catalog source of their own namespace with the OG annotation. As a result, if there is a bad catalog source in the local namespace, the user cannot subscribe to any operator in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076323[*BZ2076323*]
```

Comment:
Missed the closed parenthesis. Will catch after the review.
```suggestion
* Previously, there was a bad catalog source in the openshift-marketplace. Because of this, all subscriptions were blocked. With this update, if there is a bad catalog source in the openshift-marketplace, users can subscribe to an operator from a quality catalog source of their own namespace with the OG annotation. As a result, if there is a bad catalog source in the local namespace, the user cannot subscribe to any operator in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076323[*BZ2076323*])
```

Reply:
This looks like a namespace, based on the bug. Also, I'm pretty sure they mean original for OG.

```suggestion
* Previously, there was a bad catalog source in the `openshift-marketplace` namespace. Because of this, all subscriptions were blocked. With this update, if there is a bad catalog source in the `openshift-marketplace` namespace, users can subscribe to an operator from a quality catalog source of their own namespace with the original annotation. As a result, if there is a bad catalog source in the local namespace, the user cannot subscribe to any operator in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076323[*BZ2076323*])
```